crates/emmylua_ls/src/handlers/initialized/client_config/default_config.rs

```diff
@@ -0,0 +1,63 @@
+use std::time::Duration;
+
+use log::info;
+use serde_json::Value;
+
+use crate::{context::ServerContextSnapshot, util::time_cancel_token};
+use emmylua_code_analysis::file_path_to_uri;
+
+use super::ClientConfig;
+
+pub async fn get_client_config_default(
+    context: &ServerContextSnapshot,
+    config: &mut ClientConfig,
+    scopes: Option<&[&str]>,
+) -> Option<()> {
+    let workspace_folders = context
+        .workspace_manager
+        .read()
+        .await
+        .workspace_folders
+        .clone();
+    let main_workspace_folder = workspace_folders.get(0);
+    let client = &context.client;
+    let scope_uri = main_workspace_folder.map(|p| file_path_to_uri(p).unwrap());
```

Comment:
Note: this rolls back a change [made in previous commit](https://github.com/EmmyLuaLs/emmylua-analyzer-rust/commit/9464063492bd577b8fdbf9f985bde64928278aee#diff-a24411316d4e6cbe920f2b6f662b7d2a5e84c24077749245a2133a534da4f95fR22-R24). This is because NeoVim doesn't require workspace root to run EmmyLua. In this case, `workspace_folders` will be empty, but we should still request configs from the client.

Let me know if I've missed something and I shouldn't change this.